proxy/proxy.go

```diff
@@ -290,7 +290,9 @@ func New(r *routing.Routing, options Options, pr ...PriorityRoute) *Proxy {
 		Routing:              r,
 		Flags:                Flags(options),
 		PriorityRoutes:       pr,
-		CloseIdleConnsPeriod: -time.Second})
+		CloseIdleConnsPeriod: -time.Second,
+		OpenTracer:           &ot.NoopTracer{},
```

Comment:
i think we should move the setting of the default tracer to the WithParams function. Something like:

```
params.Tracer == nil {
   params.Tracer = &ot.NoopTracer{}
}
```